source/content/modules-known-issues.md

```diff
@@ -311,6 +311,13 @@ ___
 **Solution**: You can try to patch the [permission check in the module](https://github.com/thephpleague/oauth2-server/blob/e184691ded987c00966e341ac09c46ceeae0b27f/src/CryptKey.php#L51). The alternative is to use off-site key management tools like [Lockr](https://www.drupal.org/project/lockr)
 ___
 
+## [Update Manager](https://www.drupal.org/docs/8/core/modules/update-manager) - Drupal 8 (core)
+
+**Issue**: Sometimes, after a fresh system install, the "+ Install new module" and "+ Install new themes" buttons are missing.
```

Comment:
How about linking to the D.o issue?  https://www.drupal.org/project/drupal/issues/3033480)

That way users could get more context if they want...and maybe more importantly help fix it, or  in the future follow the link and see that it's been fixed and let us know to update our docs! =)

Reply:
Added link.